lxd/instance_post.go

```diff
@@ -874,17 +874,17 @@ func instancePostClusteringMigrate(s *state.State, r *http.Request, srcPool stor
 	return run, nil
 }
 
-// instancePostClusteringMigrateWithCeph handles moving a ceph instance from a source member that is offline.
+// instancePostClusteringMigrateWithRemoteStorage handles moving a remote shared storage instance from a source member that is offline.
 // This function must be run on the target cluster member to move the instance to.
-func instancePostClusteringMigrateWithCeph(s *state.State, r *http.Request, srcPool storagePools.Pool, srcInst instance.Instance, newInstName string, newMember db.NodeInfo, stateful bool) (func(op *operations.Operation) error, error) {
+func instancePostClusteringMigrateWithRemoteStorage(s *state.State, r *http.Request, srcPool storagePools.Pool, srcInst instance.Instance, newInstName string, newMember db.NodeInfo, stateful bool) (func(op *operations.Operation) error, error) {
 	// Sense checks to avoid unexpected behaviour.
-	if srcPool.Driver().Info().Name != "ceph" {
-		return nil, fmt.Errorf("Source instance's storage pool is not of type ceph")
+	if !srcPool.Driver().Info().Remote {
+		return nil, fmt.Errorf("Source instance's storage pool is not remote shared storage")
 	}
 
 	// Check this function is only run on the target member.
 	if s.ServerName != newMember.Name {
-		return nil, fmt.Errorf("Ceph instance move when source member is offline must be run on target member")
+		return nil, fmt.Errorf("remote shared storage instance move when source member is offline must be run on target member")
```

Comment:
Thanks for this!

Please could you make the error start with a capital letter. Thanks!

Reply:
@tomponline 
I fix it in commit https://github.com/canonical/lxd/pull/13934/commits/21dca1bcfae71290ccb83de22bab21d8f9740f94
Thanks! 😄

Reply:
@heat1024 can you please squash both commits together? 
Just so that the first commit doesn't add the lowercase error message and the second one changes it to start with a capital letter.

Reply:
Thanks please can you squash this into the first commit (helps with back porting into our lts series)